scripts/acq_sweep.py

```
@@ -148,6 +176,71 @@ def analyze(args):
+    (name, seed) trials finished by ALL arms for an apples-to-apples table, and
+    (3) compare every arm to base on the trials they BOTH finished (paired by
+    (name, seed)): median paired speedup, median energy gap, win/tie/loss at
+    --eps, and a two-sided sign test on wins-vs-losses.
+    """
+    import numpy as np
+    from scipy.stats import binomtest
+
+    df = _per_trial(args)
+    arms = sorted(df.config.unique())
+    fin = df[~df.censored].copy()
+    fin["key"] = list(zip(fin.name, fin.seed))
```

Ruff (0.15.18)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)